src/components/AstroPiModel/AstroPiModel.scss

```diff
@@ -2,7 +2,6 @@
 @import '../../editor-spacing.scss';
 
 .sense-hat {
-  height: fit-content;
```

Comment:
My only question on this would be, does it affect the web component in any way? I guess the reason it was so small (way too small as you say) was because we were trying to get the model + control panel to fit in the height. Just wondering whether this will change the way this is rendered for Astro Pi...?

Reply:
It seems ok locally? 
![Screenshot 2023-02-03 at 12 42 49](https://user-images.githubusercontent.com/14238047/216606577-4db4d313-b978-46c2-a82d-7a018fc47643.png)